Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6178,6 +6178,22 @@ ReadableByteStreamAPIEnabled:
     WebKit:
       default: false
 
+RemoteMediaSessionManagerEnabled:
+  type: bool
+  status: testable
```

Comment:
`testable` means "enabled in test infrastructure". Is that what we want, or is `unstable` more accurate?

Reply:
Oops, indeed we do!